admin/sql/updates/20231005-edit-data-idx-link-type.sql

```diff
@@ -0,0 +1,17 @@
+\set ON_ERROR_STOP 1
+
+BEGIN;
+
+DROP INDEX CONCURRENTLY IF EXISTS edit_data_idx_link_type;
+
+CREATE INDEX CONCURRENTLY edit_data_idx_link_type ON edit_data USING GIN (
```

Comment:
What are we using `CONCURRENTLY` in update scripts? (We are not using it when creating the database.)

Reply:
It allows the script to be run on the production database without downtime (without locking the table against writes). But it's not useful when creating the database indeed, since it performs extra work for no benefit in that case.